src/id.md

```diff
@@ -42,11 +46,11 @@ Follow these steps to set up an identity:
 
 Once the transaction is completed your account will show an identity!
 
-## Request Identity Judgment
+## Requesting Identity Judgment
 
 Once you have finished following all the steps outlined in the [Register an identity](/id.md#register-an-identity) section, you can request an additional verification to obtain a prominent green checkmark next to your account name. This verification serves as an indication that you are a recognized and verified user, allowing others to easily identify and trust your account.
 
-1. Request judgement from a registrar on polkadot.js.
+### 1. Request judgement from a registrar on polkadot.js.
```

Comment:
In the preceding section, the steps lack the heading decoration. Consistent formatting style needs to be applied.

Reply:
The steps in the preceding section are quite short and do not have the same sub-instructions as this one. I have made the requested changes for your consideration.